src/main/java/to/us/resume_builder/business/export_LaTeX/ResumeTemplate.java

```diff
@@ -52,18 +59,20 @@
      * @param templateName The name of the template to load.
      */
     ResumeTemplate(String templateName) {
-        String templateDirectory = ApplicationConfiguration.getInstance().getString("templates.directory");
+        final Logger LOGGER = Logger.getLogger(ResumeTemplate.class.getName());
```

Comment:
Aren't loggers supposed to be static? Not a big deal, though.

Reply:
You can't access static variables from the constructors of enums. Very weird thing, I know

Reply:
Huh. Learn something new every day.